config/locales/de.yml

```diff
@@ -341,6 +342,7 @@ de:
         one: 'Konnte %{resource} nicht speichern: ein Fehler.'
         other: 'Konnte %{resource} nicht speichern: %{count} Fehler.'
   entry_year: Einreisejahr
+  find_client: KlientIn finden
```

Comment:
Klient/in is a matter of taste. But anyhow: It is a requirement from AOZ, so not our choice....